airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -40,6 +47,14 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
                                 final StandardSyncInput syncInput,
                                 final UUID connectionId) {
 
+    final StandardCheckConnectionInput sourceConfiguration = new StandardCheckConnectionInput().withConnectionConfiguration(syncInput.getSourceConfiguration());
+    System.out.println(sourceConfiguration);
+    StandardCheckConnectionOutput sourceCheckOutput = checkActivity.run(jobRunConfig, sourceLauncherConfig, sourceConfiguration);
```

Comment:
I'm struggling to create the sStandardCheckConnectionInput`'d properly. Logging the objects, they **seem** to be structured correctly:

```
io.airbyte.config.StandardCheckConnectionInput@552cec38[connectionConfiguration={"seed":-1,"count":2000}]
```

Which matches what we use in [workers/temporal/check/connection/CheckConnectionWorkflowImpl.java](https://github.com/airbytehq/airbyte/pull/12423/files#diff-2ec3dbf62fa364de003db0507589d2d472682a646abb728e3368cf99b85f214c)

But, there's an error:

```
airbyte-worker      | 2022-04-27 22:19:41 WARN i.t.i.a.POJOActivityTaskHandler(activityFailureToResult):307 - Activity failure. ActivityId=2102143d-f794-305b-aa72-de959fd4f1a3, activityType=Run, attempt=1
airbyte-worker      | io.temporal.common.converter.DataConverterException: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "connectionConfiguration" (class io.airbyte.config.ResourceRequirements), not marked as ignorable (4 known properties: "cpu_request", "memory_request", "cpu_limit", "memory_limit"])
airbyte-worker      |  at [Source: (byte[])"{"connectionConfiguration":{"seed":-1,"count":2000}}"; line: 1, column: 29] (through reference chain: io.airbyte.config.ResourceRequirements["connectionConfiguration"])
airbyte-worker      | 	at io.temporal.common.converter.JacksonJsonPayloadConverter.fromData(JacksonJsonPayloadConverter.java:84) ~[temporal-sdk-1.8.1.jar:?]
airbyte-worker      | 	at io.temporal.common.converter.DefaultDataConverter.fromPayload(DefaultDataConverter.java:135) ~[temporal-sdk-1.8.1.jar:?]
airbyte-worker      | 	at io.temporal.common.converter.DataConverter.arrayFromPayloads(DataConverter.java:104) ~[temporal-sdk-1.8.1.jar:?]
```